Olapic/CarbonClientServiceProvider
# Support Silex 2.x

What does this Pull Request Do?
-------------------------------
- Support Silex 2.x

How should this be manually tested?
-----------------------------------
Check https://github.com/Olapic/APIv2/pull/502 

Any background context you want to provide?
-------------------------------------------
No

What are the relevant tickets?
------------------------------
No

Any extra info?
---------------
No
